src/features/testing/workflows/passive.md

```diff
@@ -0,0 +1,77 @@
+# Passive Type Workflows
+
+_For general documentation on utilizing the Workflows tab - click [here](../workflows.md)._
+
+---
+
+`Passive Workflows` take **requests** or **responses** as input. Their execution occurs in the "background" as you conduct your testing, extending the efficiency of your process.
+
+## Using Passive Workflows
+
+---
+
+Passive Workflows are **automatically triggered** when their specifications/conditions are met. These specifications/conditions are set by [Nodes](../../../concepts/nodes.md) and include prerequisites such as:
+
+- If the request/response is within a set [Scope](../../overview/scope.md).
+- If the request/response is a match according to [HTTPQL](../../../concepts/httpql.md) syntax.
+- If the prior Node's specification/condition evaluated to True or False (_Boolean value_).
+
+## Creating a New Passive Workflow: Applying Color to In-Scope GET Requests Workflow Example
+
+---
+
+_In this example - the Workflow created will color highlight GET requests within the HTTP History tab if they are within the Project's Scope._
+
+> Navigate to the **Workflow Editor** for the Passive type by following these steps:
+>
+> - Select the `Workflow` tab from the left-hand menu within the Caido window.
+> - Select the `Passive` tab.
+> - Click `+ New Workflow`.
+
+1. Enter an arbitrary name for your Workflow.
+2. (_Optional_) Enter a description of the workflow.
+3. Click `Save`.
+
+<img alt="Passive set color example." src="/_images/passive_setcolor_example.png"/>
+
+4. Click, hold and drag the `In Scope`, `Matches HTTPQL`, `If/Else` and `Set Color` Nodes into the pane directly right.
+
+> _NOTE: The_ `On intercept request` _and_ `Passive End` _Nodes are already included by default. Passive Workflows do not require an End Node in order to function properly. However, it is recommended that you include it for flow readability._
+
+5. Drag the Nodes into the top-down heirachical structure displayed. Connect them together by making Node `Connections`.
+
+## Node Relationship Explanation: : Applying Color to In-Scope GET Requests Workflow Example
+
+---
+
+_The flow of the example Workflow provided above is described below:_
```

Comment:
The next part is very dense / intense to read, it could be easier if we had more screenshots at each step of the way I think. Essentially you re-take the screenshot from `passive_setcolor_example` but click on each node and take a screenshot, highlight the node.
What do you think?

Reply:
Was going to ask this actually. Will do!